lib/active_job/retry/fixed_delay_retrier.rb

```diff
@@ -0,0 +1,49 @@
+require 'active_job/retry/fixed_options_validator'
+
+module ActiveJob
+  module Retry
+    class FixedDelayRetrier
+      def initialize(options)
+        FixedOptionsValidator.new(options).validate!
+        @retry_limit      = options.fetch(:limit, 1)
+        @retry_delay      = options.fetch(:delay, 0)
+        @fatal_exceptions = options.fetch(:fatal_exceptions, [])
+        @retry_exceptions = options.fetch(:retry_exceptions, nil)
+      end
+
+      def should_retry?(attempt, exception)
+        return false if retry_limit_reached?(attempt)
+        return false unless retry_exception?(exception)
+        true
+      end
+
+      def retry_delay(_, _)
+        @retry_delay
+      end
+
+      private
+
+      attr_reader :retry_limit, :fatal_exceptions, :retry_exceptions
+
+      def retry_limit_reached?(attempt)
+        return true if retry_limit == 0
+        return false if retry_limit == -1
+        attempt >= retry_limit
+      end
+
+      def retry_exception?(exception)
+        return true if retry_exceptions.nil? && fatal_exceptions.empty?
```

Comment:
Do you need this line? Would be nice to restructure `retry_exceptions` and `fatal_exceptions` so they're always arrays.